openmc/mgxs/library.py

```diff
@@ -354,16 +354,16 @@ def add_to_tallies_file(self, tallies_file, merge=True):
 
         Parameters
         ----------
-        tallies_file : openmc.TalliesFile
-            A TalliesFile object to add each MGXS' tallies to generate a
+        tallies_file : openmc.Tallies
+            A Tallies object to add each MGXS' tallies to generate a
```

Comment:
Perhaps "Tallies collection" in place of "Tallies object"? Depends on what you wrote for `Materials`, but either is fine with me as long as they are consistent.